sdk/README.md

```diff
@@ -0,0 +1,157 @@
+# <img src="https://app.gmx.io/favicon/apple-icon-144x144.png" width="28" height="28"> GMX SDK
+
+## Install
+
+```bash
+yarn add @gmx-io/sdk # or
+npm install --save @gmx-io/sdk
+```
+
+## Usage
+
+```typescript
+import { GmxSdk } from "@gmx/sdk";
+import { useWallet } from "wagmi";
+
+const sdk = new GmxSdk({
+  chainId: 42161,
+  rpcUrl: "https://arb1.arbitrum.io/rpc",
+  oracleUrl: "https://arbitrum-api.gmxinfra.io",
+  walletClient: useWallet().walletClient,
+  subgraph: {
+    subsquid: "https://gmx.squids.live/gmx-synthetics-arbitrum/graphql",
```

Comment:
having subsquid inside subgraph seems incorrect
they are two parallel entities, both having equal role in our platform

idk yet if it's just README typo or intended that way

Reply:
Agree, I just copied structure of subgraph urls from UI config, but here it doesn't make sense, we use only subsquid in SDK, so I just rename the object-field to `subsquidUrl` property